qiskit_quantinuum/quantinuumjob.py

```diff
@@ -334,6 +334,29 @@ def _process_results(self):
         results = []
         self._status = JobStatus.DONE
         for i, res_resp in enumerate(self._experiment_results):
+            if type(res_resp) is JobStatus:
```

Comment:
This works fine but I think a `isinstance()` would be a bit more robust here.

```suggestion
            if isinstance(res_resp, JobStatus):
```

Reply:
Done! Also, the lint check should pass, now.